MahakPal/Day1/01.c

```diff
@@ -0,0 +1,43 @@
+#include<stdio.h>
+
+int main()
+{
+    char name[50];
+    printf("Hellow! What's your name?");
+    scanf("%s", name);
+    printf("welcome,%s!\n", name);
+    // Answer: MahakPal // 
+    
+    char solution2[200];
+    //Problem second//
+    printf("Why do you want to learn C?\n ");
+    scanf("%s", solution2);
```

Comment:
%s will only store single word as input, what if you give whole statement as answer. Try it, it will give error.